### ISSUE TITLE
Message about non-completed tasks is confusing on failure

### ISSUE THREAD

**Issue by stof** [gulpjs/gulp-cli#203]
I'm using `gulp.parallel` to run several tasks in parallel. When the build succeed, everything is fine and gulp is happy with tasks completing. But when one of the tasks is failing (due to an error in one of my sass files for instance), I get a confusing output, after the plugin error, saying that some tasks failed to complete (basically all other parallel tasks, unless they were already done) and the question `Did you forget to signal async completion?`.

I know that this message is meant to report issues with sync tasks. But it seems to me that the implementation should take into account the case of failures, where it is entirely normal than some tasks don't complete, when `gulp.parallel` was used (maybe `gulp.series` suffers from it as well).
